ui/site/src/powertip.ts

```diff
@@ -537,13 +537,12 @@ class TooltipController {
   }
 
   resetPosition(element: Cash) {
-    let priorityList;
-
     if (this.options.smartPlacement) {
-      priorityList = smartPlacementLists[this.options.placement!];
-
+      let priorityList = smartPlacementLists[this.options.placement!];
+      if ($as<WithTooltip>(element).classList.contains('challenge-powertip'))
+        priorityList = [...priorityList, 's']; // so that 's' is used in case all are incorrectly judged as collisions on phones
```

Comment:
this makes the general powertip support aware of challenges, and their positioning in the lobby UI. That's a SoC leak